assemblies/prd-ce/pom.xml

```diff
@@ -329,6 +343,20 @@
       <groupId>pentaho</groupId>
       <artifactId>pentaho-metaverse-api</artifactId>
       <scope>compile</scope>
+      <exclusions>
```

Comment:
Per the jira description (https://hv-eng.atlassian.net/browse/PPP-5166)

```
/pentaho-server/pentaho-solutions/system/kettle/plugins/metaverse-plugin/lib/jackson-databind-2.14.2.jar (Can be excluded)
Metaverse has dependencies on jackson-databind, cannot be excluded here, needs to be upgraded.
```
![ppp_5166_metaverse_jackson_databind_description](https://github.com/user-attachments/assets/0e2a6385-e50b-48e4-bab0-93e77176cc59)



Is the Jira description still valid and/or is jackson-databind still in the lib folder?

Reply:
metaverse is dependent of jackson libraries during compile time and runtime. That is why it is upgraded. https://github.com/pentaho/pentaho-metaverse/pull/698/files#diff-8d04401f1cc51365fe3e32f019cd720135ba920a1a7da7f19e9c9208478701fcR90

Although it isnt present in the metaverse-plugin jar as I can see. I have to look into that. 

But as for prd-ce assembly, jackson-databind is not needed. So excluded.

Reply:
I understand your conclusion for prd-ce.